components/speedreader/rust/lib/src/readability/src/statistics.rs

```diff
@@ -388,9 +388,9 @@ this text is countedthis text is counted
         assert_eq!(326, features.moz_score_all_linear);
         // 23.2361 = sqrt(40) + sqrt(286)
         assert_approx_eq!(23.2361, features.moz_score_all_sqrt, 1e-4);
-        // 2.4495 = sqrt(286 - 280)
-        // Note, the first <p> is ignored since it has len < 280
-        assert_approx_eq!(2.4495, features.moz_score, 1e-4);
+        // 2.4495 = sqrt(286 - 140)
+        // Note, the first <p> is ignored since it has len < 140
+        assert_approx_eq!(12.0830, features.moz_score, 1e-4);
```

Comment:
The comment here is not fully updated (`2.4495` is meaningless now).

Any reason not to just put the math inline? i.e. `assert_approx_eq!(((286 - 140) as f32).sqrt(), ...)`

Reply:
updated comments and put all the math inline